vendor-extra/LiberoPageBundle/src/EventListener/InfoBarListener.php

```diff
@@ -21,10 +20,9 @@ final class InfoBarListener
 
     private $urlGenerator;
 
-    public function __construct(TranslatorInterface $translator, UrlGeneratorInterface $urlGenerator)
```

Comment:
Having second thoughts about this, might be useful to generate the path to the image?